pandas/core/window/ewm.py

```diff
@@ -518,7 +518,9 @@ def cov_func(x, y):
                 x_array,
                 start,
                 end,
-                self.min_periods,
+                # error: Argument 4 to "ewmcov" has incompatible type
+                # "Optional[int]"; expected "int"
+                self.min_periods,  # type: ignore[arg-type]
```

Comment:
The EWM subclass ensures this is set to `int` before passing to the parent class which has `Optional[int]` typing